pkg/workers/exec/konnector.go

```diff
@@ -55,6 +55,7 @@ type KonnectorMessage struct {
 	FolderToSave   string `json:"folder_to_save"` // FolderToSave is the identifier of the folder
 	AccountDeleted bool   `json:"account_deleted,omitempty"`
 
+	Version string `json:"version"`
```

Comment:
I think it should be removed (just a remaining line of code from a reverted commit), no?

Reply:
Yep, nice find ;)